playbook/app/pb_kits/playbook/pb_textarea/_textarea.tsx

```diff
@@ -31,14 +33,16 @@ type TextareaProps = {
   name?: string,
   required?: boolean,
   rows?: number,
-  resize: 'none' | 'both' | 'horizontal' | 'vertical' | 'auto',
+  resize: "none" | "both" | "horizontal" | "vertical" | "auto",
   onChange?: InputCallback<HTMLTextAreaElement>,
 }
```

Comment:
This is great! adding & Global Props after the types definition is also good practise to make sure each kit accepts our Global Props without throwing a type error. [Here's an example of that](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/pb_avatar/_avatar.tsx#L21)